DisCatSharp/Clients/DiscordClient.Events.cs

```diff
@@ -260,6 +260,18 @@ public sealed partial class DiscordClient
 		remove => this._guildIntegrationsUpdated.Unregister(value);
 	}
 	private AsyncEvent<DiscordClient, GuildIntegrationsUpdateEventArgs> _guildIntegrationsUpdated;
+
+	/// <summary>
+	/// Fired when a guild audit log entry was created.
```

Comment:
Add note to event that view audit log is required as permission for the bot to receive the event